cumulusci/robotframework/pageobjects/PageObjects.py

```diff
@@ -23,24 +25,43 @@ class PageObjects(object):
     cache = {}
 
     def __init__(self, *args):
-        BuiltIn().log("initializing PageObjects...", "DEBUG")
+        logger.info("initializing PageObjects...", "DEBUG")
         importer = robot.utils.Importer()
 
         for file_path in args:
             try:
                 importer.import_class_or_module_by_path(os.path.abspath(file_path))
-                BuiltIn().log("imported page object {}".format(file_path), "DEBUG")
+                logger.debug("imported page object {}".format(file_path))
             except Exception as e:
-                BuiltIn().log(str(e), "WARN")
+                logger.warn(str(e))
         self.current_page_object = None
 
         # Start with this library at the front of the library search order;
         # that may change as page objects are loaded.
-        BuiltIn().set_library_search_order("PageObjects")
+        try:
+            BuiltIn().set_library_search_order("PageObjects")
+        except RobotNotRunningError:
+            pass
```

Comment:
This except clause and the one below aren't covered by the tests. Are they really needed?

Reply:
You are correct. I must've added that before I figured out how to mock the context. Or something. I removed the except blocks and all of the tests still pass.